### ISSUE TITLE
Add possibility to publish to multiple platforms

### ISSUE THREAD

**Issue by SpyrexDE** [primodotso/primo-server#24]
I would like to be able to publish to Vercel **and** GitHub at the same time.
The UI even looks like it would be possible by having those "Add" and "Remove" options for each host, but it is not.

**Comment by mateomorris** [comment]
@SpyrexDE you can do that by publishing to GitHub, then manually creating a project in Vercel and connecting it to the repo. Updates will automatically be deployed to the Vercel project when the repo updates. 

Primo **could** be made to publish to both, but then publishing would take twice as long due to the upload time.

**Comment by SpyrexDE** [comment]
You are right but I think this should either be possible or the user interface should be redesigned to communicate that you are only able to set one publishing host.

**Comment by mateomorris** [comment]
That's a good point. I'll leave this open as a reminder to change that when I get a chance.
